docs/preview/continuous-patching/README.md

```diff
@@ -0,0 +1,477 @@
+
+### Floating Tagging
+
+How it works
+
+A single mutable tag, -patched, will always reference the latest patched version of your image. For instance, if your base image tag is python:3.11, one patch creates python:3.11-patched. Subsequent patches will keep overwriting that same -patched tag with new digests.
+
+  
+![PatchingTimelineExample](./media/patching_timeline_example1.png)
+
+### Which Should I Use?
+
+Incremental (**default**): Great for environments where auditability and rollbacks are critical, since each new patch is clearly identified with an unique tag.
+
+Floating: Ideal if you prefer a single pointer to the latest patch for your CI/CD pipelines. Reduces complexity by removing the need to update references in downstream applications per patch, but sacrifices strict versioning, making it difficult to rollback. 
+
+
+## Installing the Continuous Patching Workflow
+
+Download the [private CLI extension](https://acrcssc.z5.web.core.windows.net/acrcssc-1.1.1rc6-py3-none-any.whl) for Continuous patching 
+
+Run the following command:
+```sh
+az extension add --source <path to the CLI extension>
+```
+
+## Enable the Continuous Patching Workflow
+To enable Continuous Patching, follow the series of steps below that outline the CLI process. These guidelines detail the lifecycle of a continuous patching workflow, from its creation to subsequent updates to eventual deletion. 
+1. Login to Azure CLI with az login
+```sh
+az login
+```
+2. Login to ACR 
+```sh
+az acr login -n <myRegistry>
+```
+3. Use your preferred text editor to create a file named continuouspatching.json. Linux example below
+```sh
+vim continuouspatching.json //any file creation command and file name will do
+```
+Copy in the JSON Schema:
+```sh
+{
+    "version": "v1",
+    "tag-convention" : "<incremental|floating>",
+    "repositories": [{
+        "repository": "<Repository Name>",
+        "tags": ["<comma-separated-tags>"],   
+        "enabled": <true|false>
+    }] 
+}
+```
+The schema ingests specific repositories and tags in an array format. Each variable is defined below:
+
+- "version" allows the ACR team to track what schema version you’re on. Do not change this variable unless instructed to.
+- "tag-convention" this is an optional field. Allowed values are "incremental" or "floating" - refer to **Key Concepts** for more information.
+
+- "repositories" is an array that consists of all objects that detail repository and tag information
+    - "repository" refers to repository name
+    - "tags" is an array of tags separated by commas. The wildcard "*" can be used to signify all tags within that repository
+    - "enabled" is a Boolean value of true or false determining if the specified repo is on or off
+
+The following details an example configuration for a customer who wants to patch all tags (use the * symbol) within the repository “python”, and to patch specifically the “jammy-20240111” and “jammy-20240125” tags in the repository “ubuntu”. 
```

Comment:
In developer documentation, please avoid the use of smart quotes (basically, slanted single and double quotes `‘ ’` and `“ ”`). Smart quotes are not `ASCII` characters (they are Unicode characters instead), causing problems in the terminal if they are pasted as they are not considered valid opening and closing quotes. Let us change these in this line as customers may directly copy and paste these.

Also, they should be in inline code blocks.